hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/placement/algorithms/TestSCMContainerPlacementRackAware.java

```diff
@@ -563,6 +574,69 @@ public void testvalidateContainerPlacementSingleRackCluster() {
     assertEquals(0, stat.misReplicationCount());
   }
 
+  @ParameterizedTest
+  @MethodSource("decommissionNodeStates")
```

Comment:
Nit: can use existing method from `NodeStatus`:

https://github.com/apache/ozone/blob/698e2f4ffc2fd60a6164ffd541e7400d4fa2a727/hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/NodeStatus.java#L60-L62

as already used in:

https://github.com/apache/ozone/blob/698e2f4ffc2fd60a6164ffd541e7400d4fa2a727/hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/replication/TestRatisContainerReplicaCount.java#L212-L214

(`MethodSource` allows any collection (does not have to use `Stream`), and for single parameter we can use the type directly, no need to wrap i n`Arguments`.)

```suggestion
  @MethodSource("org.apache.hadoop.hdds.scm.node.NodeStatus#decommissionStates")
```

Reply:
> (MethodSource allows any collection (does not have to use Stream), and for single parameter we can use the type directly, no need to wrap i nArguments.)

Thanks @adoroszlai for the info. I didn't know that.